src/app/modules/organisations/organisations-list/organisations-list.component.html

```diff
@@ -19,7 +19,8 @@
       <div class="list-menu">
         <mat-form-field class="custom-form-field">
           <mat-label>Search for Organisation</mat-label>
-          <input matInput #searchInput>
+          <!-- <input matInput #searchInput> -->
+          <input  [(ngModel)] = "search" [appDebounceDirective] = "search" (debounceClick)="debounceMethod()" #searchInput matInput>
```

Comment:
why can't you directly call getOrganisationList function instead of debounceMethod

Reply:
Modified